aws/cloudformation/ami-manager.js

```diff
@@ -0,0 +1,112 @@
+var response = require('cfn-response');
+/** Takes an AWS CloudFormation stack name and instance id and returns the newly-created AMI ID. **/
+exports.handler = function (event, context) {
+  console.log("REQUEST RECEIVED:\n", JSON.stringify(event));
+
+  var stackName = event.ResourceProperties.StackName;
+  var instanceId = event.ResourceProperties.InstanceId;
+  var instanceRegion = event.ResourceProperties.Region;
+
+  var responseStatus = "FAILED";
```

Comment:
The cfn-response module actually provides explicitly-enumerated strings for the response status, so I will use those directly